python/lsst/obs/ctio0m9/ingest.py

```diff
@@ -138,23 +143,25 @@ def _translate_filter(self, val):
             return conversion[val]
         else:
             self.log.warn('Unmapped filter type %s found when translating filter', val)
-            return 'UNKNOWN_FILTER' #avoiding using None, as this is an alias for clear/no_filter
+            # avoiding using None, as this is an alias for clear/no_filter
+            return 'UNKNOWN_FILTER'
 
     def translate_filter(self, md):
         """Generate the standardised composite name of the two filters.
 
         Map the filters used to a standard name, and concatenate as a string,
-        so that flats can be generated for each filter pair easily. Individual filter
-        name sanitisation is done by the _translate_filter() function so it only
-        has to be defined in one place.
+        so that flats can be generated for each filter pair easily. Individual
+        filter name sanitisation is done by the _translate_filter() function
+        so it only has to be defined in one place.
 
         @param[in] md image metadata
         @return sanitized and concatenated filter name
         """
         filt1 = self._translate_filter(md.getScalar("FILTER1").rstrip().lstrip())
         filt2 = self._translate_filter(md.getScalar("FILTER2").rstrip().lstrip())
         sorted_filter = [filt1, filt2]
-        sorted_filter.sort() #we want to be insensitive to filter order, for now at least
```

Comment:
You are right that I didn't have to shift the comment to its own line. The reason I did so is personal preference. For longer commentary I prefer explaining the reasoning before the code line happens, rather than having the reason as an addendum.